addons/binding/org.openhab.binding.enocean/README.md

```diff
@@ -0,0 +1,128 @@
+
+## Supported Things
+
+This binding is developed on and tested with the following things
+
+ * USB300 and EnOceanPi gateways as OpenHab bridges
+ * The following Eltako actuators:
+    * FSR14 (light switch)
+    * FSB14 (rollershutter)
+    * FUD14 (dimmer)
+    * FSSA-230V (smart plug)
+ * The following Opus actuators:
+    * GN-A-R12V-SR-4 (light)
+    * GN-A-R12V-MF-2 (light)
+    * GN-A-R12V-LZ/UD (dimmer)
+    * GN-A-R12V-JRG-2 (rollershutter)
+    * GN-A-U230V-JRG (rollershutter)
+    * OPUS-FUNK PLUS Jalousieaktor 1fach UP (rollershutter)
+    * OPUS-Funk PLUS Steckdosenleiste (smart multiple socket)
+ * NodOn Smart Plug (ASP-2-1-10), In Wall Switch (SIN-2-2-00, SIN-2-1-0x), Permundo PSC234 (smart plug with metering)
+ * Thermokon SR04 room control
+ * Hoppe SecuSignal window handles
+ * Rocker switches (NodOn, Eltako FT55 etc)
+
+However because of the standardized EnOcean protocol it is more important which EEP this binding supports: F6-02, F6-10, D5-00, A5-10, A5-38 (switching and dimming), A5-37 (use generic things). Hence if your device supports one of these EEPs the chances are good that your device is also supported by this binding.
+
+## Discovery
+
+Most of the EnOcean devices can be automatically created and configured as an OpenHab thing through the discovery service. The EnOcean protocol defines a so called "teach-in" process to announce the abilities and services of an EnOcean device and pair devices. To pair an EnOcean device with its OpenHab thing representation, you have to differentiate between sensors and actuators.
+
+### Sensors
+
+To pair a sensor with its thing, you first have to start the discovery scan for this binding in PaperUI. Then press the "teach-in" button of the sensor. The sensor sends a teach-in message which contains the information about the EEP and the device Id of the sensor. If the EEP is known by this binding the thing representation of the device is created. The corresponding channels are created dynamically, too. 
+
+### Actuators
+ 
+If the actuator supports UTE teach-ins, the corresponding thing can be created and paired automatically. First you have to start the discovery scan for this binding in PaperUI. Then press the teach-in button of the actuator. 
+
+If the actuator does not support UTE teach-ins, you have to create, configure and choose the right EEP of the thing manually. It is important to link the teach-in channel of this thing. Afterwards you have to activate the pairing mode of the actuator. Then switch on the teach-in item(/channel) to send a teach-in message to the actuator. If the pairing was successful, you can control the actuator and unlink the teach-in channel.
+
+
+## Adding Enocean-Things (Actuators, Switches, Sensors, etc.)
+
+https://github.com/fruggy83/enocean/wiki#how-to-add-an-enocean-thing-actuator-sensor-wallswitch-etc-via-paperui
+
+## Thing Configuration
+
+Things can and should by configured through PaperUI. Following the most important config parameters:
```

Comment:
I updated the documentation regarding this point. Using PaperUI is particularly needed for the pairing process. However after pairing is done, you are able to configure your things in files.

Just a side note: I think that parameters with multiple options cannot be set in the thing file (see [#6146](https://github.com/eclipse/smarthome/issues/6146)). Implemented a workaround for this in my binding, but should be solved in ESH.

Reply:
Please correct "PaperUI" to "Paper UI".

> Using PaperUI is particularly needed for the pairing process.

Ok, understood for the devices that need pairing - but if I am not mistaken, most sensors (without security) do not require a pairing process, so you should formulate it in a way so that it is clear that it is only relevant for pairing; probably best to have a dedicated section on pairing and leave the "thing configuration" section independent of whether DSLs or UIs are used.